migration/20210803123705_stocks.sql

```diff
@@ -0,0 +1,22 @@
+-- +goose Up
+-- +goose StatementBegin
+CREATE TABLE IF NOT EXISTS stocks(
+    id int not null unique AUTO_INCREMENT,
```

Comment:
there can be multiple unique key in a table. mention explicitly about primary key.